Docs (Chore): Provide instructions on where to find gRPC Endpoint (#53)
dgraph-io/cloud-docs: content/admin/import-export.md

It is possible to import data into a Dgraph Cloud backend using [live loader](https://dgraph.io/docs/deploy/#live-loader). In order to import data, do the following steps:

1. First import your schema into your Dgraph Cloud backend, using either the [Schema API](/admin/schema) or via [the Schema Page](https://cloud.dgraph.io/_/schema).

> 2. Login to Dgraph Cloud, and find the `gRPC Endpoint` for your backend on the Settings page. This will look like `frozen-mango.grpc.us-west-1.aws.cloud.dgraph.io:443`

Reviewer: ```suggestion
2. Log into Dgraph Cloud, and find your backend's `gRPC Endpoint` on the Settings page. This will look like `frozen-mango.grpc.us-west-1.aws.cloud.dgraph.io:443`
```
